README.md

```diff
@@ -8,14 +8,12 @@
 ## Table of Contents
 
 - [Background](#background)
-- [Install](#install)
-- [Usage](#usage)
 - [Contributing](#contributing)
 - [License](#license)
 
 ## Background
 
-The Mattermost AI Copilot Plugin adds functionality for local (self-hosted) and third-party (vendor-hosted) LLMs within Mattermost v9.6 and above.
+The Mattermost AI Copilot Plugin adds functionality for local (self-hosted) and third-party (vendor-hosted) LLMs within Mattermost v9.6 and above. Visit the [Usage](docs/usage.md) page to learn about potential AI workflows with this Mattermost integration.
```

Comment:
Do we even need to keep the usage doc in this repo, considering these features are described in the product orientation? I don't think we need a document like this, and I don't want us to have two sources to maintain.

Reply:
whoops! I missed a link that we can safely remove now. I'll get that sorted. Great catch, @azigler!